src/components/Nav/Nav.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import SideBar from './SideBar';
+import wepickerLogo from '../../images/Nav/wepicker_logo.png';
+import myPage from '../../images/Nav/human.png';
+import cart from '../../images/Nav/cart.png';
+import wish from '../../images/Nav/heart.png';
+import './Nav.scss';
+
+class Nav extends Component {
+    constructor() {
+        super();
+        this.state = {
+            topLine: 'true',
+            bottomLine: 'true',
+            sideMenuBar: 'true'
```

Comment:
true를 string데이터타입으로 주시는 걸까요? 
boolean타입이라면 ' ' 을 제거해주세요!
```suggestion
            topLine: true,
            bottomLine: true,
            sideMenuBar: true
```